src/Cms/Pages.php

```diff
@@ -295,11 +296,9 @@ public function findOpen(): Page|null
 	 * Custom getter that is able to find
 	 * extension pages
 	 *
-	 * @param string $key
 	 * @param mixed $default
-	 * @return \Kirby\Cms\Page|null
 	 */
-	public function get($key, $default = null)
+	public function get(string $key, $default = null): Page|null
```

Comment:
We could get rid of the remaining docblock type as well (supported since PHP 8.0):

```suggestion
	public function get(string $key, mixed $default = null): Page|null
```